apps/06.1Checkers.js

```diff
@@ -8,11 +8,73 @@ var rl = readline.createInterface({
 });
 
 
-function Checker() {
+//we need to pass in a string into this class function
+//value will be 'white' or 'black'
+//only one of these two words
+function Checker(color) {
   // Your code here
+  //property called symbol
+  //but i cant just set it outright, if the color is white set it to String.fromCharCode(0x1<unicode>)
+  //if color is black set the symbol to String.fromCharCode(0x125CF)
+  if(color === 'white'){
+   this.symbol = String.fromCharCode(0x125CB);
+  }
+  else{
+    this.symbol = String.fromCharCode(0x125CF);
+  }
 }
 
 function Board() {
+  //create propery called checkers and assign it to an empty array
+//create method called 'createCheckers'
+this.checkers = [];
+this.createCheckers = function(){
+var whitePositions = [
+[0, 1], [0, 3], [0, 5], [0, 7],
+[1, 0], [1, 2], [1, 4], [1, 6],
+[2, 1], [2, 3], [2, 5], [2, 7]
+] ;
+var blackPositions = [
+[5, 0], [5, 2], [5, 4], [5, 6],
+[6, 1], [6, 3], [6, 5], [6, 7],
+[7, 0], [7, 2], [7, 4], [7, 6]
+] ;
+//need to put something in these above positions
+for (var i = 0; i < whitePositions.length; i++){
```

Comment:
The code inside your function should not be flush with it's declaration of the function (ex: function Board())